CMakeLists.txt

```diff
@@ -2139,7 +2159,42 @@ if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
 
 endif()
 
-if(UNIX AND NOT APPLE)
+
+if(APPLE)
+  if(QT5_TYPE STREQUAL "STATIC_LIBRARY")
+    find_package(FFTW REQUIRED)
```

Comment:
What is this needed for?

Reply:
Chromaprint and Rubberband, unfortunately setting INTERFACE_LINK_LIBRARIES is not sufficient.